src/visitor/block_transform_visitor.rs

```diff
@@ -61,11 +56,14 @@ impl VisitMut for BlockTransformVisitor<'_> {
         }
 
         let mut ident_provider = DefaultIdentProvider::new(&self.config.local_var_prefix);
-        expr.visit_mut_children_with(&mut get_visitor(
-            &mut ident_provider,
-            &self.config.csi_methods,
-            self.transform_status,
-        ));
+        let mut operation_visitor = OperationTransformVisitor {
```

Comment:
Why `Box::new` is not necessary now?

Reply:
`Box` was needed because get_visitor<'a>() method returned different instances depending on whether plus_operator is enabled or not. And rust compiler doesn't know the size at compile time so you have to box it.